diffsky/experimental/photometry_interpolation.py

```diff
@@ -1,8 +1,8 @@
 """
 """
-from dsps.sed import calc_ssp_weights_sfh_table_lognormal_mdf
+from dsps.sed.ssp_weights import calc_ssp_weights_sfh_table_lognormal_mdf
 from jax import vmap, jit as jjit
-from dsps.cosmology import age_at_z
```

Comment:
This import statement works as written with dsps v0.3

Reply:
In the version of diffsky that I had, neither of these imports worked. The code was:
from dsps import age_at_z
from dsps import calc_ssp_weights_sfh_table_lognormal_mdf
I had to make the same modifications that are now in the latest version of diffsky, so these changes are redundant. Sorry, I didn't realize that the update had already happened.